src/subdomains/supporting/log/log.service.ts

```diff
@@ -48,4 +48,25 @@ export class LogService {
 
     return this.logRepo.findOneBy({ id: Equal(maxId) });
   }
+
+  async getBankLogs(batchIds: string[]): Promise<Log[]> {
+    const query = this.logRepo
+      .createQueryBuilder('log')
+      .where('subsystem = :subsystem', { subsystem: 'UploadBank' })
+      .andWhere('severity = :severity', { severity: LogSeverity.INFO });
+
+    query.andWhere(
+      new Brackets((query) =>
+        batchIds.forEach((id, index) => {
+          if (index === 0) {
+            query.where(`log.message LIKE '%${id}%'`);
+          } else {
+            query.orWhere(`log.message LIKE '%${id}%'`);
```

Comment:
Hm, I think the best way to do this is probably
`query.where('log.message LIKE :message', { message: '%${id}%'});`